common/changes/@microsoft/api-extractor/updateApiModel_2019-06-06-21-47.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Generate ApiTypeParameter entries and type alias types",
+      "type": "patch"
```

Comment:
I've changed this to `minor`.  We're aiming to keep the versions the same between the packages.  

(@iclanton maybe the best way would be to enable Rush lockstep versioning for **api-extractor**, **api-extractor-model**, and **api-documenter**.)